statwrap/ux.py

```diff
@@ -0,0 +1,84 @@
+'''
+User Interactibility
+'''
+
+import ipywidgets as widgets
+import io
+import pandas as pd
+from IPython.display import display
+from IPython import get_ipython
+
+
+class DataUploadWidget:
```

Comment:
I think it'd be better to make this a subclass of widgets.VBox with children uploader, button, output. 

This can have a few advantages which I can write out or explain on a call. I'd also deprioritize the auto_display functionality (because it won't display if you run `x = widgets.VBox()`. We can create a convenience function the user might call instead of actually calling `DataUploadWidget` directly to handle some of those aspects.